docs/configuration copy.md

```diff
@@ -0,0 +1,132 @@
+---
+description: Telliot tweaks and settings to keep your rig running smoothly.
+---
+
+# Configuration reference
+
+## CLI reference
+
+Telliot commands and config file options are as the following:
+
+#### Required Flags <a id="docs-internal-guid-d1a57725-7fff-a753-9236-759dd3f42eed"></a>
+
+* `--config` \(path to your config file.\)
+
+#### Telliot Commands
+* `--logConfig` \(location of logging config file; default path is current directory\)
```

Comment:
Is this file generated?

Why is this here we don't use `logConfig` anymore?

Reply:
Oh, thanks! this file should be removed :+1: